### DISCUSSION TITLE
Solidus usable as an marketplace platform like eBay or Gumtree?

### DISCUSSION THREAD

**Discussion by lifeboy** [solidusio/solidus#3892] [Troubleshooting]
I'm looking for a platform that will allow members to upload their own product to sell.  Could I do that with solidus?

**Comment by seandawes** [comment]
Yes you could check out some example stores who do this like https://bevv.com/

**Comment by kennyadsl** [comment]
Hey @lifeboy, yes you can! We also have a work in progress plugin here https://github.com/boomerdigital/solidus_marketplace that should give you a basic idea on how to change the core to make it work. We are also starting to build an integration with PayPal Commerce Platform Marketplace capabilities that will take advantage of its built-in marketplace functionality like payouts and merchant onboarding from within the platform.

**Reply by Sankalp-Dwivedi** [reply]
Hey @kennyadsl  https://github.com/boomerdigital/solidus_marketplace is no longer in active development , so is there any other gem or any document or blog that can assist in developing a market place using solidus?

**Reply by kennyadsl** [reply]
You can take a look at https://github.com/solidusio-contrib/solidus_paypal_marketplace, which is another option, based on the PayPal Commerce Platform payment gateway. It's probably not up to date with the latest version of Solidus, but I think it's not hard to make it work. If PayPal is an option, I'd give it a try (I can try to help if needed).

**Comment by dhonig** [comment]
I wouldn't say that its not under active development.  Quite alot of good
work was done on this gem.  If there is interest in using it then it could
be brought up to current.

**Reply by fthobe** [reply]
Hey, are you currently still operating any marketplace running solidus?

**Comment by BenMorganIO** [comment]
@Sankalp-Dwivedi

Hey there, I was the technical cofounder of bevv.com and we're working on retromarket.co ATM. It's very possible to use Solidus as a marketplace solution. For Retro Market, we've created a `Retro::StoreOrder` model to encapsulate all logic relevant to order management for stores. This way there is no need for exposing the whole `Spree::Order` model and you can expose only the data that you feel is appropriate for the store user. If you have any specific marketplace questions, I answer them here or on the Slack.

**Reply by fthobe** [reply]
Hey, are you currently still operating any marketplace running solidus?

**Comment by lifeboy** [comment]
No, I never did.  I just explored the possibility at some stage.

Roland
